commcare_connect/opportunity/helpers.py

```diff
@@ -308,17 +313,28 @@ def get_opportunity_list_data(organization, program_manager=False):
     )
 
     if program_manager:
+        total_deliveries_sq = Subquery(
+            CompletedWork.objects.filter(opportunity_access__opportunity_id=OuterRef("pk"))
+            .values("opportunity_access__opportunity_id")
+            .annotate(total=Sum("saved_completed_count", distinct=True))
+            .values("total")[:1],
+            output_field=IntegerField(),
+        )
+
+        verified_deliveries_sq = Subquery(
+            CompletedWork.objects.filter(opportunity_access__opportunity_id=OuterRef("pk"))
+            .values("opportunity_access__opportunity_id")
+            .annotate(total=Sum("saved_approved_count", distinct=True))
+            .values("total")[:1],
```

Comment:
This question is relevant to all the subqueries, but do these error if there are no completed works?